src/components/Footer.jsx

```diff
@@ -1,5 +1,14 @@
 const Footer = () => {
-  return <button className="footer">Contact us</button>;
+  const handleScrollToTop = () => {
+    window.scrollTo({ top: 0, behavior: "smooth" });
+  };
+
+  return (
+    <div className="footer">
+      <button onClick={handleScrollToTop}>Back to top</button>
```

Comment:
Love this as an idea, you have quite a lot of empty space at the bottom of the page on this one that would be worth looking at.